app/services/create_fund_activity.rb

```diff
@@ -21,9 +22,13 @@ def call
     activity.accountable_organisation_reference = "GB-GOV-13"
     activity.accountable_organisation_type = "10"
 
-    activity.extending_organisation = Organisation.find_by!(service_owner: true)
-
     activity.save(validate: false)
     activity
   end
+
+  private
+
+  def service_owner
+    Organisation.find_by_service_owner(true)
```

Comment:
@tahb @MariluzHerrera @lozette : Thinking out load/question: Could `service_owner` be a class method on `Organisation` - it's like a constant and I keep looking for a nicer way to express it, my undertanding is that then we could do things like:

````
activity.extending_organisation = Organisation.service_owner
````
which is quite nice in my mind…

Reply:
I think it would be nice to be able to say that however whilst it is technically possible to have more than one organisations marked as a `service_owner` then I think this approach would risk implying there is more security than there is and lead to unexpected problems in the future.

Getting to a state where the system can only have a single service owner would be great. It seems like we need a way to represent a new type of resource that quacks in almost entirely the same way as an organisation - `ServiceOwner`? Perhaps single table inheritance _would be_ appropriate as a way to help with this one?